src/gui/base/TextField.ts

```diff
@@ -201,7 +213,9 @@ export class TextField implements ClassComponent<TextFieldAttrs> {
 				".flex-grow.rel",
 				autofillGuard.concat([
 					m("input.input" + (a.alignRight ? ".right" : ""), {
-						autocomplete: a.preventAutofill ? "off" : "",
+						autocomplete: a.preventAutofill
```

Comment:
I think we can remove preventAutofill from the attrs and just have set the AutocompleteValues.off in all places that need it

Reply:
that doesn't work sadly afair

Reply:
I meant that we can have the a.autocompleteValue === .off instead of the preventAutofill as a marker for our workarounds

Reply:
I agree, I misunderstood, my bad

Reply:
yes that makes sense